sources/system/file-system/win32-file-system.dylan

```diff
@@ -626,3 +626,9 @@ define function %root-directories
     end
   end
 end function %root-directories;
+
+define function %create-symbolic-link
+    (target :: <microsoft-directory-locator>, link :: <microsoft-directory-locator>)
+ => ()
+  win32-file-system-error("symlink", "create symbolic link. Not supported in Win32")
```

Comment:
It should be enough to just call `error` here; the `win32-file-system-error` function retrieves the Win32 equivalent of `errno` and includes that in the `error` message.